tests/utils.go

```diff
@@ -1554,6 +1580,35 @@ func GetRunningPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, names
 	return pod
 }
 
+func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance) *k8sv1.Pod {
```

Comment:
Here we have a function called `GetRunningPodByVirtualMachineInstance` which you can use instead.

Reply:
The problem is that we never expect the pod to actually run, since there are no hosts that satisfy the runtime class.

Reply:
Good point. :+1: